app/packages/files/file.ts

```diff
@@ -44,10 +50,16 @@ export default abstract class File {
     shouldShowTags = false;
     shouldShowRevisions = true;
     providerHandlesVersioning = true;
+    owner: unknown;
 
     currentUser: CurrentUserService;
+    @service intl!: Intl;
+    @service toast!: Toast;
+
 
-    constructor(currentUser: CurrentUserService, fileModel: FileModel) {
+    constructor(owner: unknown, currentUser: CurrentUserService, fileModel: FileModel) {
+        setOwner(this, owner);
```

Comment:
drive-by question: could you copy the owner off `fileModel` instead of adding an arg to all these classes?
```
setOwner(this, getOwner(fileModel));
```
?

Reply:
Mmmmaybe? I'll take a look.